about.html

```diff
@@ -12,62 +12,108 @@
+      <div class="headinfo">
+        <div class="head left">
+            <a href="./index.html"><img class="logo" src="img/lambda-black.png" alt="Great Idea! Company logo." /></a></br>
+        </div>
+        <div class="head right">
+          <a class="head links" href="./index.html" >Home</a> 
+          <a class="head links" href="./about.html" >About</a> 
+          <a class="head links" href="" >Product</a> 
+          <a class="head links" href="" >Blog</a> 
+          <a class="head links" href="./contact.html" >Contact</a>
+        </div>
+    </div>
+    <div class="head image">
+      <img class="logo" src="img/jumbo-about.png" alt="jumbo image" /></br>
+    </div>
+    <div class="head image">
+    <h1> About </h1>
+      <p>
+        Leverage agile frameworks to provide a robust synopsis for high level 
+        overviews. Iterative approaches to corporate strategy foster collaborative 
+        thinking to further the overall value proposition. Organically grow the 
+        holistic world view of disruptive innovation via workplace diversity and 
+        empowerment.
+      </p>
+    </div>
+    <div class="line"></div>
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
+      <div class="columns">
+        <div class="column content">
+        <img src="img/about-plan.png" alt="strategy">
+        <h1>
+          Strategy
+        </h1>
+        Iterative approaches to corporate strategy foster collaborative thinking 
+        to further the overall value proposition.
+
+        <button class="abutton">
+        Learn More
+        </button>
+      </div>
+      <div class="column content">
+        <img src="img/about-working.png" alt="strategy">
+        <h1>
+          How We Work
+        </h1>
+        <p>
+        Iterative approaches to corporate strategy foster collaborative thinking 
+        to further the overall value proposition.
+      </p>
+        <button class="abutton">
+        Learn More
+        </button>
+      </div>
+    </div>
+
+    <div class="columns">
+        <div class="column content">
+            <img src="img/about-office.png" alt="strategy">
+        <h1>
+          Places We Work
+        </h1>
+        Iterative approaches to corporate strategy foster collaborative thinking 
+        to further the overall value proposition.
+
+        <button class="abutton">
+        Learn More
+        </button>
+      </div>
+      <div class="column content">
+          <img src="img/about-meeting.png" alt="strategy">
+        <h1>
+          Collaboration
+        </h1>
+        <p>
+        Iterative approaches to corporate strategy foster collaborative thinking 
+        to further the overall value proposition.
+      </p>
+        <button class="abutton">
+        Learn More
+        </button>
+      </div>
+    </div>
+
+
+  <div>
+    <h1>
     Let's Work Together
-
+  </h1>
```

Comment:
No Div to hold this h1 and p tags together?

That's why you couldn't easily style them?
The only flaw on your project!
Please correct and commit.